src/main/java/seedu/address/model/common/DatePastPredicate.java

```diff
@@ -13,7 +13,7 @@ public DatePastPredicate() { }
 
     @Override
     public boolean test(Date date) {
-        return nowDate.isBefore(date.getDate());
+        return nowDate.isBefore(date.getDate()) || nowDate.isEqual(date.getDate());
```

Comment:
wait ah I think we are trying to allow today to be added? like today is 2021-03-29, it should be allowed as a deadline?